.gitlab/ci/build_quartz.yml

```diff
@@ -18,7 +18,7 @@ allocate_resources_build_quartz:
   extends: .on_quartz
   stage: allocate_resources
   script:
-    - salloc -N 1 -c 36 -p pdebug -t 30 --no-shell --job-name=${SLURM_ALLOC_NAME}
+    - salloc -N 1 -c 36 -p pdebug -t 30 --no-shell --job-name=${PROJECT_ALLOC_NAME}
```

Comment:
30 minutes is definitely not going to work.. The TPL builds on genie take 10/12/20 minutes respectively when I ran them last night.  The Serac build and test takes less than a minute each.

Reply:
I allocate 1 node (36 cores) for 30 mins, then I divide it between the 3 jobs, so each get 30 mins.

Reply:
This is not a fix for execution time in #48 btw.